restcomm/restcomm.extension.controller/src/main/java/org/restcomm/connect/extension/controller/ExtensionController.java

```diff
@@ -116,4 +117,55 @@ public ExtensionResponse executePostOutboundAction(final IExtensionRequest er, L
         //TODO: implement actual calls
         return response;
     }
+
+    public ExtensionResponse executePreInboundAction(final IExtensionRequest er, List<RestcommExtensionGeneric> extensions) {
+        ExtensionResponse response = new ExtensionResponse();
```

Comment:
what is the meaning of this default response?, how do we detect if this response is coming from an actual extension or not? maybe the empty response is good enough for the framework, just wondering...

Reply:
@jaimecasero The `ExtensionResponse` object will be set by each extension. The `response` holds an object that can be used to pass data from one extension back to RC, such as custom headers, custom proxy headers etc.

Currently, we don't use the `post-*-action` methods so we just create a `response` object and return it.

The idea for the `post-*-action` is to use them for maintenance and logistic, such as update logs, update db, notify services (charging) etc.....